modules/firelens_log_config/main.tf

```diff
@@ -0,0 +1,136 @@
+# This is just a module to provide a template for creating a log configuration for an ECS task definition.
+
+locals {
+  tld = var.region == "eu-central-1" ? "eu" : "com"
+
+  ecs_shared_env_vars = <<EOF
+        { "name" : "ENVIRONMENT", "value" : "${var.environment_name}" },
+        { "name" : "APP_NAME", "value" : "${var.app_name}" }
+EOF
+
+  fire_lens_container = <<EOF
+  {
+    "essential": true,
+    "image": "906394416424.dkr.ecr.${var.region}.amazonaws.com/aws-for-fluent-bit:stable",
+    "name": "log_router",
+    "cpu": 0,
+    "user": "0",
+    "environment": [],
+    "volumesFrom": [],
+    "portMappings": [],
+    "mountPoints": [],
+    "firelensConfiguration": {
+      "type": "fluentbit",
+      "options": {
+        "enable-ecs-log-metadata": "true"
+      }
+    },
+    "memoryReservation": 50
+  }
+EOF
+
+datadog_ecs_agent_task_def = <<EOF
```

Comment:
Noticed this task def along with the `fire_lens_container` above it. I don't think we need these also in this module if we're adding it separately to the other modules as well and this module is only for the `firelens_log_config`. Unless there's a way to have them all here and remove them from the other modules. Might save some extra lines. Ex: https://github.com/smartrent/nerves-hub-terraform/pull/25/files#diff-577383bb530ff11610a70333091158c1cbb3daadde7de344d67c7609415df780R41

Reply:
Just tested this as I had been aiming towards removing it from the other modules and it is now in dev. I'll remove from all the module PRs and leave here

Reply:
We can - what would you like?

Reply:
We could go with something longer but more specific like `datadog_and_firelens_log_config` or something more generic like `logging_configs` that covers both things I think since datadog does logging 🤷‍♀️. Either one works for me. Or if you have a preference.

Reply:
Renamed to `logging_configs` ✅